src/Interactive/EditorFeatures/Core/Implementation/Interactive/Commands/HelpCommand.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.Interactive;
+using Microsoft.VisualStudio.InteractiveWindow;
+using Microsoft.VisualStudio.InteractiveWindow.Commands;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Interactive
+{
+    [Export(typeof(IInteractiveWindowCommand))]
+    [InteractiveWindowRole(InteractiveWindowRoles.Any)]
+    internal sealed class HelpReplCommand : InteractiveWindowCommand
+    {
+        internal const string CommandName = "help";
+
+        private static readonly string[] s_details = new[]
+        {
+            "  command-name    Name of the REPL command to display help on.",
+        };
+
+        public override string Description
+        {
+            get { return "Display help on specified REPL command, or all available REPL commands and key bindings if none specified."; }
+        }
+
+        public override IEnumerable<string> Names
+        {
+            get { yield return CommandName; }
+        }
+
+        public override string CommandLine
+        {
+            get { return "[command-name]"; }
+        }
+
+        public override IEnumerable<string> DetailedDescription
+        {
+            get { return base.DetailedDescription; }
```

Comment:
@amcasey, no I think the override of ```DetailedDescription``` is completely unnecessary here.  Removed (and in duplicate, original source).